### PR TITLE
[2.8] Update wrangler imports to v2

### PR DESCRIPTION
**Pull request by KevinJoiner**
### Backport of: #44206

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,8 +2,12 @@ module github.com/rancher/rancher
 
 go 1.20
 
-// on release remove this wrangler replace and use the latest tag
-replace github.com/rancher/wrangler v1.1.1 => github.com/rancher/wrangler v1.1.1-0.20230831050635-df1bd5aae9df
+replace (
+	github.com/rancher/aks-operator v1.2.0 => github.com/KevinJoiner/rancher-aks-operator v0.0.0-20240124143758-627d80adf4e6
+	github.com/rancher/channelserver v0.5.1-0.20230719220800-0a37b73c7df8 => github.com/KevinJoiner/channelserver v0.0.0-20240124134223-b6d3819b0a98
```
Comment (KevinJoiner) [inline go.mod:7]: Remove and update commit after https://github.com/rancher/channelserver/pull/25 is merged.

File: go.mod [hunk 2]
```diff
@@ -2,8 +2,12 @@ module github.com/rancher/rancher
 
 go 1.20
 
-// on release remove this wrangler replace and use the latest tag
-replace github.com/rancher/wrangler v1.1.1 => github.com/rancher/wrangler v1.1.1-0.20230831050635-df1bd5aae9df
+replace (
+	github.com/rancher/aks-operator v1.2.0 => github.com/KevinJoiner/rancher-aks-operator v0.0.0-20240124143758-627d80adf4e6
```
Comment (KevinJoiner) [inline go.mod:6]: Remove and update commit when https://github.com/rancher/aks-operator/pull/375 is merged.

File: go.mod [hunk 3]
```diff
@@ -2,8 +2,12 @@ module github.com/rancher/rancher
...
+	github.com/rancher/channelserver v0.5.1-0.20230719220800-0a37b73c7df8 => github.com/KevinJoiner/channelserver v0.0.0-20240124134223-b6d3819b0a98
+	github.com/rancher/eks-operator v1.3.0 => github.com/KevinJoiner/rancher-eks-operator v0.0.0-20240123183955-8f2f749b3ed1
```
Comment (KevinJoiner) [inline go.mod:8]: Remove and update commit when https://github.com/rancher/eks-operator/pull/335 is merged.

File: go.mod [hunk 4]
```diff
@@ -2,8 +2,12 @@ module github.com/rancher/rancher
...
+	github.com/rancher/gke-operator v1.2.0 => github.com/KevinJoiner/rancher-gke-operator v0.0.0-20240124144130-9c4adac1def8
```
Comment (KevinJoiner) [inline go.mod:9]: Remove and update commit when https://github.com/rancher/gke-operator/pull/297 is merged.

File: pkg/apis/go.mod [hunk 5]
```diff
@@ -3,8 +3,12 @@ module github.com/rancher/rancher/pkg/apis
 go 1.20
 
 replace (
-	// wrangler bracnhes need to be updated before replace can be removed
-	github.com/rancher/wrangler v1.1.1 => github.com/rancher/wrangler v1.1.1-0.20230831050635-df1bd5aae9df
+	github.com/rancher/aks-operator v1.2.0 => github.com/KevinJoiner/rancher-aks-operator v0.0.0-20240124143758-627d80adf4e6
+	github.com/rancher/eks-operator v1.3.0 => github.com/KevinJoiner/rancher-eks-operator v0.0.0-20240123183955-8f2f749b3ed1
+	github.com/rancher/gke-operator v1.2.0 => github.com/KevinJoiner/rancher-gke-operator v0.0.0-20240124144130-9c4adac1def8
```
Comment (KevinJoiner) [inline pkg/apis/go.mod:8]: Remove and update commit when https://github.com/rancher/gke-operator/pull/297 is merged.

File: pkg/apis/go.mod [hunk 6]
```diff
@@ -3,8 +3,12 @@ module github.com/rancher/rancher/pkg/apis
 go 1.20
 
 replace (
-	// wrangler bracnhes need to be updated before replace can be removed
-	github.com/rancher/wrangler v1.1.1 => github.com/rancher/wrangler v1.1.1-0.20230831050635-df1bd5aae9df
+	github.com/rancher/aks-operator v1.2.0 => github.com/KevinJoiner/rancher-aks-operator v0.0.0-20240124143758-627d80adf4e6
+	github.com/rancher/eks-operator v1.3.0 => github.com/KevinJoiner/rancher-eks-operator v0.0.0-20240123183955-8f2f749b3ed1
```
Comment (KevinJoiner) [inline pkg/apis/go.mod:7]: Remove and update commit when https://github.com/rancher/eks-operator/pull/335 is merged.

File: pkg/apis/go.mod [hunk 7]
```diff
@@ -3,8 +3,12 @@ module github.com/rancher/rancher/pkg/apis
 go 1.20
 
 replace (
-	// wrangler bracnhes need to be updated before replace can be removed
-	github.com/rancher/wrangler v1.1.1 => github.com/rancher/wrangler v1.1.1-0.20230831050635-df1bd5aae9df
+	github.com/rancher/aks-operator v1.2.0 => github.com/KevinJoiner/rancher-aks-operator v0.0.0-20240124143758-627d80adf4e6
```
Comment (KevinJoiner) [inline pkg/apis/go.mod:6]: Remove and update commit when https://github.com/rancher/aks-operator/pull/375 is merged.

File: go.mod (hunk 2 above)
Comment (mjura) [reply go.mod:6]: It was merged, please rebase

File: go.mod (hunk 3 above)
Comment (mjura) [reply go.mod:8]: It was merged, please rebase

File: go.mod (hunk 4 above)
Comment (mjura) [reply go.mod:9]: It was merged, please rebase

File: pkg/apis/go.mod (hunk 5 above)
Comment (mjura) [reply pkg/apis/go.mod:8]: It was merged, please rebase

**Comment by mjura** [comment]
@KevinJoiner please rebase, when are you going to merge it ?

**Comment by KevinJoiner** [comment]
@mjura Sorry, I am not at Rancher anymore 😢 , so I won't be merging this.
But my man @MbolotSuse may be able to give you a hand.

**Comment by mjura** [comment]
rebase is needed

**Comment by MbolotSuse** [comment]
Closing this pr, looks like this was taken care of by #43652. Thanks @chiukapoor!

**Pull request closed without merging**